### PR TITLE
Fix php 8.0 compatibility warning

### PR DESCRIPTION
**Pull request by 8ctopus**
#81

### CODE REVIEW

**Comment by 8ctopus** [comment]
Sorry made a bit of a mess but should be all good now.

File: tests/CompositionObject/OptionUnitTest.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 namespace Slack\Tests\Object;
```
Comment (cmbuckley) [inline tests/CompositionObject/OptionUnitTest.php:2]: Just one more to update here, but everything else looks good!

Comment (8ctopus) [reply tests/CompositionObject/OptionUnitTest.php]: done

**Comment by alek13** [comment]
This will break BC, so we create `3.x-dev` alias for `master` branch.

Also create `2.x` branch for other BC changes.

I merge this PR into `master`, but will not create a tag for version `3.x` for now.

**Pull request merged**